testing-guidance/workflow-tests/physics/physics-lkg-workflows.md

```diff
@@ -83,37 +91,125 @@ Any project that has the following Gems enabled:
+|-------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Creating a Terrain Layer Spawner entity with PhysX Heightfield Collider                       | <ol><li>Add Terrain World Renderer and Terrain World components to Level prefab entity. Create an entity with Terrain Layer Spawner, Axis Aligned Box Shape, PhysX Heightfield Collider, Terrain Physics Heightfield Collider and Terrain Height Gradient list components.</li><li>Assign Terrain Height Gradient list component with a gradient.</li><li>Verify the collider and collision behaviors of generated terrain layer with other PhysX entities in Editor.</li><li> Assign Terrain Physics Heightfield Collider with various Physics Materials and verify that their values are reflected in Terrain layer spawner behavior.</li></ol>                                                                                                                                   | <ul><li>Terrain layer spawner entity with PhysX Heighfield Collider can be created in Editor.</li><li>Assigned gradient impacts the appearance of Terrain layer spawner and its collider.</li><li>Terrain layer spawner entity with assigned colliders correctly collides and interacts with PhysX entities.</li><li>Assigned Physics materials impact the collision behavior of Terrain layer spawner entity.</li></ul>                                                                                                                                                                                                                                                                                                                         |
```

Comment:
Request Step 1: For the list of components, restructure it as an unordered list so that way it's easy to visually comprehend the required components and not lose something in the sentence format.
Request Step 3: Requests should describe the behavior to be performed on the system, not a "verification" step. Reword or remove to keep requests to actionable behaviors then add any verifications to the _Things to Watch For_ column.
Request Step 4: Move the verification language to the Things to Watch For column.